main/test/ca/uwaterloo/flix/TestExamples.scala

```diff
@@ -41,20 +41,21 @@ class TestExamples extends Suites(
   //new FlixTest("the-ast-typing-problem-with-polymorphic-records", "examples/the-ast-typing-problem-with-polymorphic-records.flix")(Options.DefaultTest.copy(xallowredundancies = true)),
 
   // Others
-  new FlixTest("TestBelnap", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
-  new FlixTest("TestConstant", "examples/domains/Constant.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
-
-  new FlixTest("ConstantParity", "examples/domains/ConstantParity.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
-  new FlixTest("ConstantSign", "examples/domains/ConstantSign.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
-  new FlixTest("Interval", "examples/domains/Interval.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
-  new FlixTest("IntervalAlt", "examples/domains/IntervalAlt.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
-  // new FlixTest("IntervalInf", "examples/domains/IntervalInf.flix", "examples/domains/Belnap.flix")(compiled = false), // TODO: Broken
-  new FlixTest("Mod3", "examples/domains/Mod3.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
-  new FlixTest("Parity", "examples/domains/Parity.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
-  new FlixTest("ParitySign", "examples/domains/ParitySign.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
-  new FlixTest("PrefixSuffix", "examples/domains/PrefixSuffix.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
-  new FlixTest("Sign", "examples/domains/Sign.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
-  new FlixTest("StrictSign", "examples/domains/StrictSign.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
+  // TODO re-enable these tests after their laws are update to the new syntax
+//  new FlixTest("TestBelnap", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
+//  new FlixTest("TestConstant", "examples/domains/Constant.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
+//
+//  new FlixTest("ConstantParity", "examples/domains/ConstantParity.flix", "examples/domains/Belnap.flix")(Options.TestWithLibrary),
```

Comment:
I don't want to disable these tests (for other reasons- we refactoring the solver and need them).

Is it possible just to comment out all the `#foo# annotations? (I guess there are many?)

Reply:
There are many. Luckily, I'm becoming a master at writing little scripts to manipulate Flix syntax.